DEVDOC.md

```diff
@@ -718,7 +718,7 @@ only the (transformed) capture commands needs to be replayed.
 
 If the user-requested replay does require to transform the initial commands,
 then the pre-warm replay is abandonned and a new replay of the transformed
-inital commands and then the transformed capture commands are executed.
+inital commands and then the transformed capture commands is executed.
```

Comment:
"commands" is plural, so "are" was correct.
The last part of the sentence sounds weird, maybe we should rephrase it to "then the pre-warm replay is abandonned and a new replay of the transformed inital and capture commands are executed."

Reply:
I think the verb conjugates with "replay" here, as in: `a new replay (of <whatever>) is executed.`

Thanks for the suggestion, I applied it. But I kept "is". What do you think?

Reply:
Yes, thanks, you are right. I was confused by the initial weird sentence.